digdag-core/src/main/java/io/digdag/core/database/DatabaseProjectStoreManager.java

```diff
@@ -292,10 +290,17 @@ public StoredWorkflowDefinitionWithProject getLatestWorkflowDefinitionByName(int
         public List<StoredWorkflowDefinitionWithProject> getLatestActiveWorkflowDefinitions(
                 int pageSize,
                 Optional<Long> lastId,
+                Optional<String> name,
                 AccessController.ListFilter acFilter)
             throws ResourceNotFoundException
         {
-            return autoCommit((handle, dao) -> dao.getLatestActiveWorkflowDefinitions(siteId, pageSize, lastId.or(0L), acFilter.getSql()));
+            return autoCommit((handle, dao) -> dao.getLatestActiveWorkflowDefinitions(
+                    siteId,
+                    pageSize,
+                    lastId.or(0L),
+                    name.isPresent() && !name.get().isEmpty() ? "%" + escapeLikeParameter(name.get()) + "%" : "%",
```

Comment:
I felt it was difficult to understand what this line is trying. How about adding a comment on this line or extracting this line to a private method like `createWorkflowNamePattern()` or something?

Reply:
I will add a comment to explain the intention the index structure on `workflow_definition`, and the performance insight on the SQL statement of `dao. getLatestActiveWorkflowDefinitions`

Reply:
Thank you for your suggestion.
- I refactored here  at af05bb1
- I left a comment about the performance and the index used in this SQL statement at 
983b31a